app/_hub/kong-inc/jwt/index.md

```diff
@@ -298,6 +298,12 @@ or as cookie, if the name is configured in `config.cookie_names` (which is not e
 curl --cookie jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJhMzZjMzA0OWIzNjI0OWEzYzlmODg5MWNiMTI3MjQzYyIsImV4cCI6MTQ0MjQzMDA1NCwibmJmIjoxNDQyNDI2NDU0LCJpYXQiOjE0NDI0MjY0NTR9.AhumfY35GFLuEEjrOXiaADo7Ae6gt_8VLwX7qffhQN4 http://kong:8000/{route path}
 ```
 
+gRPC requests can include the JWT in a header:
+
+```bash
+grpcurl -H 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJhMzZjMzA0OWIzNjI0OWEzYzlmODg5MWNiMTI3MjQzYyIsImV4cCI6MTQ0MjQzMDA1NCwibmJmIjoxNDQyNDI2NDU0LCJpYXQiOjE0NDI0MjY0NTR9.AhumfY35GFLuEEjrOXiaADo7Ae6gt_8VLwX7qffhQN4' ...
```

Comment:
Is this example correct - is the jwt passed in a header like this? It looks a bit different than examples above

Reply:
it's the same as the one passed in a header (first curl example), since grpc doesn't allow uri params, and i guess apps won't carry cookies

Reply:
oops, missed the first example